mcp2210-gpio.c

```diff
@@ -93,7 +93,11 @@ int mcp2210_gpio_probe(struct mcp2210_device *dev)
 		return -EINVAL;
 
 	gpio->label		= "mcp2210";
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,5,0)
+	gpio->parent	= &dev->udev->dev;
```

Comment:
Please keep code formatting the same. We're using the same standard as Linux -- 8 spaces per tab, so `parent` needs another tab after it.